CHANGELOG.md

```diff
@@ -2,15 +2,13 @@
 
 ## Unreleased
 
-- New lint: Check for redundant words in ping names ([#355](https://github.com/mozilla/glean_parser/pull/355))
+- Add support for URL metric type ([#361](https://github.com/mozilla/glean_parser/pull/361))
```

Comment:
You will need to rebase, so we don't lose the other change message here.

Reply:
I did rebase... just did it wrong heh. I'll fix it.